### PR TITLE
Fix batch normalization equation in TF backend

### PR DESCRIPTION
**Pull request by sukolsak**
Update the batch normalization equation in the TF backend to follow the [original paper][1] and the [TensorFlow source code][2].

[1]: https://arxiv.org/abs/1502.03167
[2]: https://github.com/tensorflow/tensorflow/blob/333dc32ff79af21484695157f3d141dc776f7c02/tensorflow/python/ops/nn_impl.py#L742

### CODE REVIEW

**Comment by fchollet** [comment]
Right. Note that it's just a docstring.
